src/quo2/components/buttons/dynamic_button.cljs

```diff
@@ -0,0 +1,103 @@
+(ns quo2.components.buttons.dynamic-button
+  (:require [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.components.icon :as icon]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]))
+
+(defn- in?
+  "true if coll contains elm"
```

Comment:
in my opinion, it's better to be explicit with variable naming. I don't think there's much value add to character saving.
consider -   
```
(defn- in?
  "true if collection contains element"
  [collection element]
  (some #(= element %) collection))
```
which I assume it to be.

although perhaps this is a clojurescript practice I'm unaware of and there are some advantages to this character saving? 🤔